src/framework/musesampler/internal/musesamplerconfiguration.cpp

```diff
@@ -73,7 +74,7 @@ muse::io::path_t MuseSamplerConfiguration::userLibraryPath() const
         return io::path_t(path);
     }
 
-    io::path_t path = settings()->value(USER_MUSESAMPLER_PATH).toString();
+    io::path_t path = settings()->value(USER_MUSESAMPLER_PATH).toQString();
```

Comment:
That looks wrong

Reply:
working on update - thanks for the review

Reply:
This still seems to apply